src/components/Modals/Onboarding.vue

```diff
@@ -1,3 +1,7 @@
+<!--
+ONBOARDING Structure
+-- components
```

Comment:
What is this comment?

Reply:
deleted it because it doesnt seem to be important